src/notifications/ManagerLinux.cpp

```diff
@@ -59,6 +60,7 @@ NotificationsManager::postNotification(const mtx::responses::Notification &notif
         const auto sender   = cache::displayName(
           room_id, QString::fromStdString(mtx::accessors::sender(notification.event)));
         const auto text = utils::event_body(notification.event);
+        const auto formattedText = cmark_markdown_to_html(text.toStdString().c_str(), text.length(), CMARK_OPT_UNSAFE);
```

Comment:
You can't assume the body to be markdown.

Reply:
I think that the input bar automatically assumes input to be markdown, though? Or do you want it to only render markdown if MD is enabled in the settings?